lib/common.js

```diff
@@ -84,6 +84,7 @@ function createConfig(env = {}) {
   }
 
   const mergeLabels = parseLabels(env.MERGE_LABELS, "automerge");
+  const mergeRemoveLabels = parseLabels(env.MERGE_REMOVE_LABELS, "automerge");
```

Comment:
This also needs to be added in the return, in line 99

Reply:
Fixed.